viennacl/src/main/scala/org/apache/mahout/viennacl/opencl/javacpp/DenseColumnMatrix.scala

```diff
@@ -36,12 +36,24 @@ import org.bytedeco.javacpp.annotation._
 @Name(Array("viennacl::matrix<double,viennacl::column_major>"))
 final class DenseColumnMatrix(initDefault:Boolean = true) extends MatrixBase {
 
-  def this(nrow: Int, ncol: Int, ctx: Context = new Context()) {
+  def this(nrow: Int, ncol: Int) {
+    this(false)
+    allocate(nrow, ncol, new Context())
+  }
+
+  def this(nrow: Int, ncol: Int, ctx: Context) {
     this(false)
     allocate(nrow, ncol, ctx)
   }
 
-  def this(data: DoublePointer, nrow: Int, ncol: Int, ctx: Context = new Context(Context.MAIN_MEMORY)) {
```

Comment:
@rawkintrevo: 
```
$MAHOUT_HOME/viennacl$ mvn  test -Dscala.version=2.11.2 -Dscala.compat.version=2.11 -Pviennacl  
```
(I'd tried built with 2.11.2)
tests pass: 
```
All tests passed.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:36 min
[INFO] Finished at: 2017-06-24T13:49:59-07:00
[INFO] Final Memory: 20M/296M
[INFO] ------------------------------------------------------------------------
```
Sweet- I'd like to try this in Pseudo cluster mode as well, this afternoon.